### ISSUE TITLE
Excluding some style tags from inlining

### ISSUE THREAD

**Issue by Mefu** [jonkemp/style-data#5]
For the issue jonkemp/inline-css#56. Following addition to index.js works well enough in my project. I should admit I do not grasp how whole inlining process works, so this might not be the best place or best way to do it. 

```javascript
    ...
    $('style').each(function (index, element) {
        var mediaQueries;
        // exclude style tags with data-no-inline attribute set to true
        if($(element).data("no-inline") === true) {
            return;
        }
        styleDataList = element.childNodes;
        if (styleDataList.length !== 1) {
            callback(new Error('empty style element'));
            return;
        }
        styleData = styleDataList[0].data;
        if (options.applyStyleTags) {
            results.css.push(styleData);
        }
        if (options.removeStyleTags) {
            if (options.preserveMediaQueries) {
                mediaQueries = mediaQueryText(element.childNodes[0].nodeValue);
                element.childNodes[0].nodeValue = mediaQueries;
            }
            if (!mediaQueries) {
                $(element).remove();
            }
        }
    });
    ...
```
Should I create a pull request for this? If I should, I have few questions. Should I change name of the attribute (`data-no-inline`) ? Also which one of the following is better?
```html
<style type="text/css" data-no-inline="true">
  ...
</style>
```
or
```html
<style type="text/css" data-no-inline>
  ...
</style>
```

**Comment by jonkemp** [comment]
@Mefu I would suggest using `data-embed` to follow the same convention that juice does.

https://github.com/Automattic/juice#special-markup

**Comment by jonkemp** [comment]
Fixed 3b9a19bd5d088f0df6a5f03ea55006c560ea458b.
